modules/solid_mechanics/test/tests/shell/static/pinched_cylinder_symm_unstructured.i

```diff
@@ -1,17 +1,5 @@
 # Test for displacement of pinched cylinder (similar to pinch_cyl_symm.i)
-# The Test is run with an unstructured mesh
```

Comment:
Note that I'm removing this results table because the results differ somewhat from the prior results. I don't have the three meshes used for convergence, so these are likely not correct. This could be added back in once the follow-on commit with the cylinder coordinate system gets added.